mood-analysis.rb

```diff
@@ -28,5 +36,49 @@ def analyze_mood(words)
   "05/11 Yay, yay, yay! I'm having a awfuly great day."
 ]
 
-puts analyze_mood(text[0])
-puts analyze_mood(text[1])
+def moods(text)
+  text.each do |blog|
+    face = analyze_mood(blog)
+    puts blog[0..5] + face
+  end
+end
```

Comment:
I like that you made this a method even though I didn't require it and that you used variables to make the code more readable.